src/Project/functions/validateCompilerOptions.ts

```diff
@@ -18,7 +18,8 @@ function y(str: string) {
 	return kleur.yellow(str);
 }
 
-function validateTypeRoots(nodeModulesPath: string, typeRoots: Array<string>) {
+function validateTypeRoots(nodeModulesPath: string, rawTypeRoots: Array<string>) {
+	const typeRoots = rawTypeRoots.map(p => path.resolve(p));
```

Comment:
typeRoots is already iterated over a bit later. it's more efficient and clear to resolve there, rather than here.
also, resolve implicitly normalizes, I believe? The normalize call might not be necessary anymore.

Reply:
Yes `path.resolve` normalizes. The new solution should be much cleaner.